scimono-compliance-tests/Readme.md

```diff
@@ -0,0 +1,78 @@
+# SCIM 2.0 Compliance tests
+
+A set of integration tests that are executed against external SCIM 2.0 provider implementation and verify that if it is compliant with the SCIM 2.0 standard.
+
+## Prerequisite
+
+### Required tools
+- JDK Version 1.8
+- Maven - version 3.x
```

Comment:
Maven is not required if we distribute binaries. I think it should be listed as an option.